scripts/vhelm_run.py

```diff
@@ -0,0 +1,271 @@
+from dataclasses import dataclass
```

Comment:
Could be good to put everything into a heim folder (rename everything from vhelm to heim).